### PR TITLE
[WIP] [SDB-1087] Add /labels to ocm-api-model

### PR DESCRIPTION
**Pull request by gshilin-sdb**

### CODE REVIEW

**Comment by gshilin-sdb** [comment]
@igoihman PTAL

**Comment by igoihman** [comment]
@vkareh @nimrodshn @tzvatot PTAL

File: model/accounts_mgmt/v1/labels_resource.model [hunk 1]
```diff
@@ -0,0 +1,65 @@
+/*
+Copyright (c) 2019 Red Hat, Inc.
```
Comment (vkareh) [inline model/accounts_mgmt/v1/labels_resource.model:2]: `s/2019/2020/`

File: model/accounts_mgmt/v1/labels_resource.model [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+		// If the parameter isn't provided, or if the value is empty, then the order of the
+		// results is undefined.
+		in Order String
+
+		// Search criteria.
+		//
+		// The syntax of this parameter is similar to the syntax of the _where_ clause
+		// of an SQL statement, but using the names of the attributes of the label
+		// instead of the names of the columns of a table. For example, in order to
+	    // retrieve labels with key starting with my:
```
Comment (vkareh) [inline model/accounts_mgmt/v1/labels_resource.model:51]: Indentation/alignment here is off

**Pull request closed without merging**